test/bridge/utilities/fork.ts

```diff
@@ -0,0 +1,46 @@
+import { deployments, ethers, upgrades, network } from "hardhat"
+import {ProxyAdmin, SynapseBridge} from "../../../build/typechain";
+import {impersonateAccount} from "../../utils";
+
+// upgradeForkBridge upgrade a forked bridge to the latest version
+export async function upgradeBridgeProxy(bridgeAddress: string){
+    // set the current bridge to a SynapseBridge type so we can pull out the owner
+    const synapseBridge = await ethers.getContractFactory("SynapseBridge");
+
+    const proxyAdminContract = await ethers.getContractFactory("ProxyAdmin")
+    const proxyAdmin = (proxyAdminContract.attach(await getProxyAdmin(bridgeAddress))) as ProxyAdmin
+    const proxyAdminOwner = await proxyAdmin.owner()
+
+    const signer = await impersonateAccount(proxyAdminOwner)
+
+    // set the balance
+    await network.provider.send("hardhat_setBalance", [
+        signer._address,
+        "0x1228610962826298768",
+    ]);
+
+    // upgrade the contract
+    await proxyAdmin.connect(signer).upgrade(bridgeAddress, (await synapseBridge.deploy()).address)
+}
+
+// addBridgeOwner adds a new bridge owner to a deployede bridge contract for forking
```

Comment:
deployede -> deployed

Reply:
good catch. We should look into a spellcheck linter, we do this in go with `golang-ci.yml`